ideal-voting-server/src/main/scala/cz/idealiste/idealvoting/server/Http.scala

```diff
@@ -35,10 +36,11 @@ class Http(voting: Voting) {
             req.options.map(req => CreateOption(req.title, req.description)),
             req.voters,
           )
-          (titleMangled, token) <- voting.createElection(createElection)
+          now <- clock.currentDateTime
+          (titleMangled, token) <- voting.createElection(createElection, now)
           resp = LinksResponse(
             List(
-              Link(show"v1/election/admin/$titleMangled/$token", "election-view-admin", GET),
+              Link(show"/v1/election/admin/$titleMangled/$token", "election-view-admin", GET),
```

Comment:
@radim91 The `/` at the start of `/v1/election/...` is supposed to there, right?